tests/cases/conformance/types/uniqueSymbol/uniqueSymbolsErrors.ts

```diff
@@ -1,86 +1,86 @@
 // @target: esnext
 
 // declarations
-declare const {}: symbol();
-declare let invalidLetType: symbol();
-declare var invalidVarType: symbol();
+declare const {}: unique symbol;
+declare let invalidLetType: unique symbol;
+declare var invalidVarType: unique symbol;
 
 // function arguments and return types
-declare function invalidArgType(arg: symbol()): void;
-declare function invalidRestArgType(...arg: symbol()[]): void;
-declare function invalidReturnType(): symbol();
-declare function invalidThisType(this: symbol()): void;
-declare function invalidTypePredicate(n: any): n is symbol();
-declare function invalidTypeParameterConstraint<T extends symbol()>(): void;
-declare function invalidTypeParameterDefault<T = symbol()>(): void;
+declare function invalidArgType(arg: unique symbol): void;
+declare function invalidRestArgType(...arg: unique symbol[]): void;
+declare function invalidReturnType(): unique symbol;
+declare function invalidThisType(this: unique symbol): void;
+declare function invalidTypePredicate(n: any): n is unique symbol;
+declare function invalidTypeParameterConstraint<T extends unique symbol>(): void;
+declare function invalidTypeParameterDefault<T = unique symbol>(): void;
 
 // classes
 class InvalidClass {
-    constructor(invalidConstructorArgType: symbol()) {}
+    constructor(invalidConstructorArgType: unique symbol) {}
 
-    readonly invalidReadonlyPropertyType: symbol();
-    invalidPropertyType: symbol();
-    invalidArgType(arg: symbol()): void { return; }
-    invalidRestArgType(...args: symbol()[]): void { return; }
-    invalidReturnType(): symbol() { return; }
-    invalidThisType(this: symbol()): void { return; }
-    invalidTypePredicate(n: any): n is symbol() { return; }
-    invalidTypeParameterConstraint<T extends symbol()>(): void { return; }
-    invalidTypeParameterDefault<T = symbol()>(): void { return; }
-    get invalidGetter(): symbol() { return; }
-    set invalidSetter(arg: symbol()) { return; }
+    readonly invalidReadonlyPropertyType: unique symbol;
+    invalidPropertyType: unique symbol;
+    invalidArgType(arg: unique symbol): void { return; }
+    invalidRestArgType(...args: unique symbol[]): void { return; }

```

Comment:
This looks odd. This is meant to be an array of unique symbols, yes? But this _looks_ like a unique array of symbols. (IE, parens around `(unique symbol)[]` should be required?)